nss-day-cohort-41/react-nutshell-thepecans
# Fix react init

Description
Delete subdirectory containing React app files. Re-initialize React in the main project directory.

Type of change


 Bug fix (non-breaking change which fixes an issue)

Testing Instructions
Checkout to branch and confirm that all files are in the correct locations.

Checklist:
X My code follows the style guidelines of this project
X I have performed a self-review of my own code
X I have commented my code, particularly in hard-to-understand areas
X My changes generate no new warnings or errors
X I have added test instructions that prove my fix is effective or that my feature works
